EXP-Client/progress/ArchivedDay.js

```diff
@@ -0,0 +1,129 @@
+import React, { useState } from 'react';
+import {
+  View,
+  Text,
+  SafeAreaView,
+  ScrollView,
+  StyleSheet,
+  Pressable
+} from 'react-native';
+import Checkbox from 'expo-checkbox';
+import ProgressCircle from 'react-native-progress/Circle';
+
+function Graph({ tasks }) {
+  const completedCount = 2; // tasks.reduce(({ name, deadline, completed }, count) => completed ? count + 1 : count, 0);
+  return (
+    <View style={styles.graph}>
+      <ProgressCircle
+        size={80}
+        progress={completedCount / tasks.length}
+        showsText={true}
+        formatText={() => `${Math.round((completedCount / tasks.length) * 100)}%`}
+        style={styles.progressCircle}
+      >
+        <Text style={styles.progressText}>April 1, 2023</Text>
+      </ProgressCircle>
+    </View>
+  );
+}
+  
+function HeaderText({ tasks }) {
+  return (
+    <View style={styles.headerText}>
+      <Text style={{ fontSize: 20 }}>On this day you had...</Text>
+      {tasks.map(({ name, deadline, completed }) =>
+        <Text style={{ fontSize: 15 }} key={name}>{name}</Text>
+      )}
+    </View>
+  );
+}
+  
+export default function ArchivedDay({ tasks }) {
+  
+  const [completedCount, setCompletedCount] = useState(0); // TODO: Need to actually update the completed count
+
+  return (
+    // <Provider>
+      <View style={styles.content}>
+        <View style={styles.progress}>
+          <Graph tasks={tasks} />
+          <HeaderText tasks={tasks} />
+        </View>
+        <SafeAreaView>
+          <ScrollView bounces={false}>
+            {tasks.map((item, index) => (
+              <Pressable
+                // TODO: Actually navigate to the detailed view 
+                onPress={() => console.log('Navigate to TaskItem')}
+              >
```

Comment:
What do we mean by detailed view of the task item?

Reply:
this refers to Steven's detailed view page